app/src/main/java/at/roteskreuz/stopcorona/model/repositories/ReportingRepository.kt

```diff
@@ -300,20 +279,29 @@ class ReportingRepositoryImpl(
 
     private suspend fun uploadRevokeSuspicionInfo(temporaryExposureKeysFromSDK: List<TemporaryExposureKey>): MessageType.Revoke.Suspicion {
         return withContext(coroutineContext) {
-            val infectionMessages =
+            val yellowTemporaryExposureKeys =
                 infectionMessengerRepository.getSentTemporaryExposureKeysByMessageType(MessageType.InfectionLevel.Yellow)
-                    .map { message ->
-                        TemporaryExposureKeysWrapper(
-                            message.rollingStartIntervalNumber,
-                            message.password,
-                            message.messageType
-                        )
-                    }.asTemporaryExposureKeys(temporaryExposureKeysFromSDK)
+            val listOfYellowTemporaryExposureKeysFromSDK = yellowTemporaryExposureKeys
+                .map { temporaryExposureKey ->
+                    TemporaryExposureKeysWrapper(
+                        temporaryExposureKey.rollingStartIntervalNumber,
+                        temporaryExposureKey.password,
+                        temporaryExposureKey.messageType
```

Comment:
This would still be yellow, right?

Reply:
I see.  It's changed below. Couldn't it be set to `MessageType.GeneralRevoke` immediately? And the val named `listOfTemporaryExposureKeysToRevokeFromSDK`

Reply:
Let's fix it afterwards